### PR TITLE
add initial docblocks, fix clear errors

### PR DESCRIPTION
**Pull request by markushausammann**

### CODE REVIEW

**Comment by markushausammann** [comment]
Another easy one, now I need to go to sleep. Real improvements tomorrow.

File: src/DocRaptor.php [hunk 1]
```diff
@@ -58,52 +79,89 @@ public function setDocumentType($document_type)
...
     }
 
-    # values should be one of 'html' or 'none'
-    #   'none' - (default) skip input validation on the DR side
-    #   'html' - perform validation on the input html on the DR side and
-    #            error if non-parseable input provided
+    /**
+     * Toggle validation of HTML by DocRaptor
+     *
+     * @todo should probably just be a bool flag
```
Comment (janxious) [inline src/DocRaptor.php:105]: Could probably make it smart based on the doc type: https://docraptor.com/documentation#api_strict

File: src/DocRaptor.php [hunk 2]
```diff
@@ -58,52 +79,89 @@ public function setDocumentType($document_type)
...
+     * @param bool|string $filename
+     * @return bool|mixed
+     */
     public function fetchDocument($filename = false)
     {
         if ($this->api_key != '') {
             $url = $this->url_protocol . '://docraptor.com/docs?user_credentials=' . $this->api_key;
             $fields = array(
-                'doc[document_type]' => $this->type,
+                'doc[document_type]' => $this->document_type,
```
Comment (janxious) [inline src/DocRaptor.php:164]: facepalm

**Comment by janxious** [comment]
:+1:

**Pull request merged**